tensorrt-graphsurgeon/meta.yaml

```diff
@@ -1,9 +1,9 @@
-{% set version = "0.4.1" %}
-{% set sha256 = "03ce0427686e19d60919f6ba68dea5446a26ac53f4398bdea4ec96aec30740e7" %}    #[ppc64le]
-{% set trt_tar = "TensorRT-7.0.0.11.CentOS-7.6.ppc64le-gnu.cuda-10.2.cudnn7.6.tar.gz" %} #[ppc64le]
+{% set version = "0.4.5" %}
```

Comment:
We want these version updates in open-ce/conda_build_config.yaml too.

Reply:
yes, I have those local - will push those to the open PR there.